filebeat/docs/inputs/input-common-harvester-options.asciidoc

```diff
@@ -20,6 +20,60 @@ Here are some sample encodings from W3C recommendation:
     gbk, hz-gb-2312,
     * euc-kr, euc-jp, iso-2022-jp, shift-jis, and so on
 
+Valid encodings:
```

Comment:
The additions LGTM, but they make lines 17-21 confusing. We show encodings there that aren't shown under "valid encodings."  If it's true that some of the encodings we show are not supported, then I suggest removing lines 17 through 21 completely.